ui/src/common/actions.ts

```diff
@@ -432,22 +439,25 @@ export const StateActions = {
       `${args.name}\n\n${args.description}` :
       args.name;
     const subgroups = args.subgroups ? [...args.subgroups] : [];
+    const parentString = args.parentGroup || SCROLLING_TRACK_GROUP;
+    if (parentString === SCROLLING_TRACK_GROUP) {
+      state.scrollingTracks.push(args.id);
```

Comment:
Are we sure that Perfetto UI doesn't expect all the things in this to be tracks and not track groups? Besides, for almost all groups, their summary track has the same ID as the group, so isn't this ambiguous?

I get the feeling like we should refactor this `scrollingTracks` state out of existence by actually implementing the "scrolling tracks group" as a real `TrackGroup` to make the whole tree structure consistent.

Reply:
I changed scrollingTracks to account for Ids that represent track groups and/or tracks
```
 globals.state.scrollingTracks.forEach(
      (id) => {
        let trackLike : TrackState | TrackGroupState =
        globals.state.tracks[id];
        // If is a track
        if (trackLike && trackLike.name) {
          rootNode.push(m(TrackPanel, {key: id, id, selectable: true}));
          return;
        }
        trackLike = globals.state.trackGroups[id];
        // If is a trackGroup
        if (trackLike && trackLike.name) {
          renderGroup(trackLike, rootNode);
        }
      },
    );
    ```

Reply:
Cool, looks like that's the only other reference to it outside of the `actions.ts`. Thanks.